Source/common/new.santa.proto

```diff
@@ -0,0 +1,283 @@
+syntax = "proto3";
+
+import "google/protobuf/any.proto";
+
+option objc_class_prefix = "SNTPB";
+
+package santa;
+
+message UserInfo {
+  int32 uid = 1;
+  string name = 2;
+}
+
+message GroupInfo {
+  int32 gid = 1;
+  string name = 2;
+}
+
+message ProcessID {
+  int32 pid = 1;
+  int32 pidversion = 2;
+}
+
+message NanoTime {
+    int64 sec;
+    int64 nsec;
+}
+
+message MicroTime {
+    int64 sec;
+    int64 usec;
+}
+
+message CodeSignature {
+  bytes cdhash = 1;
+  string signing_id = 2;
+  string team_id = 3;
+}
+
+message Stat {
+  int32 dev = 1;
+  uint32 mode = 2;
+  uint32 nlink = 3;
+  uint64 ino = 4;
+  UserInfo user = 5;
+  GroupInfo group = 6;
+  int32 rdev = 7;
+  NanoTime a_time = 8;
+  NanoTime m_time = 9;
+  NanoTime c_time = 10;
+  NanoTime b_time = 11;
+  int64 size = 12;
+  int64 blocks = 13;
+  int32 blksize = 14;
+  uint32 flags = 15;
+  int32 gen = 16;
```

Comment:
Is this field likely to be useful?

Reply:
Depends on the FS. I think it's more useful with NFS (?). I don't think it's used by APFS (?).